com.emqx.MQTTX.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: com.emqx.MQTTX
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+# Electron doesn't use a traditional locale format
+# so separate-locales is useless.
+separate-locales: false
+command: mqttx.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=~/.config/MQTTX
```

Comment:
Shouldn't this happen by default? Other Electron apps properly respect `XDG_CONFIG_HOME`.

Reply:
So should I remove it from this line?

Reply:
Should be the default.